_mobile/android.md

```diff
@@ -162,7 +163,7 @@ Where `analyzeImage` method processes camera output, `android.media.Image`.
 
```

Comment:
"Where [the] analyzeImage process [the] camera output, android.media.Image. It uses [the] aforementioned..."